diff --git a/CMakeLists.txt b/CMakeLists.txt
index 20254facb..956a8f117 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -248,15 +248,15 @@ add_custom_target(doxygen-deploy
     VERBATIM
 )
 
-add_custom_target(format
-    python ${CMAKE_SOURCE_DIR}/etc/clang_format.py format
-    VERBATIM
-)
-
-add_custom_target(format-lint
-    python ${CMAKE_SOURCE_DIR}/etc/clang_format.py lint
-    VERBATIM
-)
+#add_custom_target(format
+#    python ${CMAKE_SOURCE_DIR}/etc/clang_format.py format
+#    VERBATIM
+#)
+#
+#add_custom_target(format-lint
+#    python ${CMAKE_SOURCE_DIR}/etc/clang_format.py lint
+#    VERBATIM
+#)
 
 add_custom_target(docs
     DEPENDS hugo doxygen-current
